app/src/router.js

```diff
@@ -82,6 +88,6 @@ async function featureAvailable(store, networkSlug, feature) {
 
```

Comment:
this featureAvailable logic feels hacky to me. why not just display the message on the page? is the redirect really necessary?

Reply:
I think the redirect concretely is the least complex thing in that logic. You'd need to have nearly the same code to do what you are proposing. As long as we fetch availability from the network Object, that is. 

Maybe we could do this in another way, like having a super simple switch in DB that directly connects to the Vue router :shrug: I don't know. To me it looks fine the logic as it currently is

Reply:
it's not really about complexity - though i guess if this is the simplest solution that's an ok argument. why is a redirect the best way to solve this problem? it feels like a hacky way of keeping someone away from a page.